src/main/java/ome/logic/ConfigImpl.java

```diff
@@ -246,23 +245,26 @@ public String getConfigValue(String key) {
 
     @RolesAllowed("system")
     public Map<String, String> getConfigDefaults() {
-        File etc = new File("etc");
-        File omero = new File(etc, "omero.properties");
-        Properties p = new Properties();
+        //TODO: remove hard-coded list
+        List<String> files = new ArrayList<String>();
+        files.add("omero-model.properties");
+        files.add("omero-common.properties");
+        files.add("omero-server.properties");
+        files.add("omero-blitz.properties");
+        files.add("omero.properties");
         Map<String, String> rv = new HashMap<String, String>();
-        try {
-            FileReader r = new FileReader(omero);
-            p.load(r);
-            for (Entry<Object, Object> entry : p.entrySet()) {
-                    rv.put(entry.getKey().toString(),
-                            entry.getValue().toString());
+        Iterator<String> i = files.iterator();
+        while (i.hasNext()) {
+            try (InputStream stream = getClass().getResourceAsStream(i.next())) {
+                Properties p = new Properties();
+                p.load(stream);
```

Comment:
I'm not sure what I'm missing here but in some local testing with m6 for `omero-model.properties` the `stream` is `null`.

Reply:
hummm

Reply:
If I adjust the `getClass().getResourceAsStream` to instead get the `ApplicationContext` injected and use `applicationContext.getResource(i.next()).getInputStream()` then a `stream` *is* provided that includes `omero.client.ui.menu.dropdown.colleagues.label=Members` yet for `TestConfig.testClientDefaults` `cfg.getClientConfigDefaults` returns nothing.

Reply:
I think I have figured a fix: I'll open a PR by EOB.

Reply:
Opened https://github.com/ome/omero-server/pull/33.